Global/editUserModal.php

```diff
@@ -24,6 +23,9 @@
                         <label class="custom-file-label" for="uploadImage">Choose file</label>
                     </div>
                 </div>
+                <div id="uploadImageText" hidden>
+                    <small>Invalid file type.</small>
```

Comment:
Keep it hidden as default

Reply:
Use is-invalid class (Bootstrap) instead of using CSS red

Reply:
Done